BrowserKit/Sources/Common/Logger/LoggerCategory.swift

```diff
@@ -20,6 +20,9 @@ public enum LoggerCategory: String {
     /// Related to homepage UI and it's data management.
     case homepage
 
+    /// Related to new homepage UI and it's data management for the homepage rebuild project.
+    case newHomepage
```

Comment:
I almost want to say this should be `homepage` and the old homepage stuff should be renamed `legacyHomepage` but, honestly, it's not a big deal, not even a nit, and, so, not worth the effort to make the changes.. Just mentioning this in passing because we generally use the "legacy" nomenclature when we move over.

Reply:
I agree but I do think it's worth the effort. Not just for the logs but the file names too. The word new attached to a file quickly becomes meaningless but the word legacy will always be true.

Reply:
happy to make the change! I wasn't sure when though. I originally didn't want to touch any of the homepage code, and wanted to a rename once the feature is more complete, but if it's helpful to do it earlier I can make the change.

Reply:
created new ticket here: https://mozilla-hub.atlassian.net/browse/FXIOS-10221